### ISSUE TITLE
Source URL's should be defined as anchor links.

### ISSUE THREAD

**Issue by aligoren** [apache/kibble-1#29]
Source URLs should be defined as an anchor element.

As you can see

Currently, all links in the [sources](https://demo.kibble.apache.org/organisations.html?page=sources) page are not an anchor element. All of them just a text.

May it be good, if a source URL open in a new tab when a user clicks them.

**Comment by Humbedooh** [comment]
Good idea - what about maybe an icon to the right of them for opening in a new tab? Some people might prefer the URL stays as a textNode, to prevent accidental clicks. :)
